### PR TITLE
Mobile replay beta almost over banner

### PR DESCRIPTION
**Pull request by drguthals**
## DESCRIBE YOUR PR
We'd like to let beta users know ASAP that the beta is ending soon so that we can get more people to get started NOW. On January 6th we will have additional updates in a separate PR for the GA.

## IS YOUR CHANGE URGENT?  

Help us prioritize incoming PRs by letting us know when the change needs to go live.
- [X] Urgent deadline (GA date, etc.): 
- [ ] Other deadline: 
- [ ] None: Not urgent, can wait up to 1 week+

## PRE-MERGE CHECKLIST

*Make sure you've checked the following before merging your changes:*

- [ ] Checked Vercel preview for correctness, including links
- [ ] PR was reviewed and approved by any necessary SMEs (subject matter experts)
- [ ] PR was reviewed and approved by a member of the [Sentry docs team](https://github.com/orgs/getsentry/teams/docs)

### CODE REVIEW

File: docs/platforms/android/session-replay/index.mdx [hunk 1]
```diff
@@ -7,7 +7,9 @@ description: "Learn how to enable the Beta of Mobile Session Replay in your app.
 
 <Note>
 
-Mobile support for Session Replay is in Beta. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
+Mobile support for Session Replay is in Beta until January 6th, 2025. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
```
Comment (drguthals) [inline docs/platforms/android/session-replay/index.mdx:10]:
```suggestion
Mobile support for Session Replay is in beta until January 6th, 2025. Features available in beta are still work-in-progress and may have bugs. We recognize the irony.
```

File: docs/platforms/android/session-replay/index.mdx [hunk 2]
```diff
@@ -7,7 +7,9 @@ description: "Learn how to enable the Beta of Mobile Session Replay in your app.
 
 <Note>
 
-Mobile support for Session Replay is in Beta. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
+Mobile support for Session Replay is in beta until January 6th, 2025. Features available in beta are still work-in-progress and may have bugs. We recognize the irony.
+
+All Beta users will continue to receive 2 months free of Session Replay for Mobile once we transition to GA.
```
Comment (drguthals) [inline docs/platforms/android/session-replay/index.mdx:12]:
```suggestion
All beta users will continue to receive 2 months free of Session Replay for Mobile once we transition to GA.
```

File: docs/platforms/apple/guides/ios/session-replay/index.mdx [hunk 3]
```diff
@@ -7,9 +7,11 @@ description: "Learn how to enable the Beta of Mobile Session Replay in your app.
 
 <Note>
 
-Mobile support for Session Replay is in Beta. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
+Mobile support for Session Replay is in Beta until January 6th, 2025. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
```
Comment (drguthals) [inline docs/platforms/apple/guides/ios/session-replay/index.mdx:10]:
```suggestion
Mobile support for Session Replay is in beta until January 6th, 2025. Features available in beta are still work-in-progress and may have bugs. We recognize the irony.
```

File: docs/platforms/apple/guides/ios/session-replay/index.mdx [hunk 4]
```diff
@@ -7,9 +7,11 @@ description: "Learn how to enable the Beta of Mobile Session Replay in your app.
 
 <Note>
 
-Mobile support for Session Replay is in Beta. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
+Mobile support for Session Replay is in beta until January 6th, 2025. Features available in beta are still work-in-progress and may have bugs. We recognize the irony.
 
-If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-cocoa/issues/new?assignees=&labels=Platform%3A+Cocoa%2CType%3A+Bug&projects=&template=bug.yml) with a link to a relevant replay in Sentry if possible.
+All Beta users will continue to receive 2 months free of Session Replay for Mobile once we transition to GA.
```
Comment (drguthals) [inline docs/platforms/apple/guides/ios/session-replay/index.mdx:12]:
```suggestion
All beta users will continue to receive 2 months free of Session Replay for Mobile once we transition to GA.
```

File: docs/platforms/flutter/session-replay/index.mdx [hunk 5]
```diff
@@ -9,9 +9,11 @@ description: "Learn how to enable the Mobile Session Replay Beta in your app."
 
 Flutter Session Replay is available on **iOS** and **Android**.
 
-Mobile support for Session Replay is in Beta. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
+Mobile support for Session Replay is in Beta until January 6th, 2025. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
```
Comment (drguthals) [inline docs/platforms/flutter/session-replay/index.mdx:12]:
```suggestion
Mobile support for Session Replay is in beta until January 6th, 2025. Features available in beta are still work-in-progress and may have bugs. We recognize the irony.
```

File: docs/platforms/flutter/session-replay/index.mdx [hunk 6]
```diff
@@ -9,9 +9,11 @@ description: "Learn how to enable the Mobile Session Replay Beta in your app."
 
 Flutter Session Replay is available on **iOS** and **Android**.
 
-Mobile support for Session Replay is in Beta. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
+Mobile support for Session Replay is in beta until January 6th, 2025. Features available in beta are still work-in-progress and may have bugs. We recognize the irony.
 
-If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-dart/issues/new?template=BUG_REPORT.yml) with a link to a relevant replay in Sentry if possible.
+All Beta users will continue to receive 2 months free of Session Replay for Mobile once we transition to GA.
```
Comment (drguthals) [inline docs/platforms/flutter/session-replay/index.mdx:14]:
```suggestion
All beta users will continue to receive 2 months free of Session Replay for Mobile once we transition to GA.
```

File: docs/platforms/react-native/session-replay/index.mdx [hunk 7]
```diff
@@ -7,9 +7,11 @@ description: "Learn how to enable the Beta of Mobile Session Replay in your app.
 
 <Note>
 
-Mobile support for Session Replay is in Beta. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
+Mobile support for Session Replay is in Beta until January 6th, 2025. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
```
Comment (drguthals) [inline docs/platforms/react-native/session-replay/index.mdx:10]:
```suggestion
Mobile support for Session Replay is in beta until January 6th, 2025. Features available in beta are still work-in-progress and may have bugs. We recognize the irony.
```

File: docs/platforms/react-native/session-replay/index.mdx [hunk 8]
```diff
@@ -7,9 +7,11 @@ description: "Learn how to enable the Beta of Mobile Session Replay in your app.
 
 <Note>
 
-Mobile support for Session Replay is in Beta. Features available in Beta are still work-in-progress and may have bugs. We recognize the irony.
+Mobile support for Session Replay is in beta until January 6th, 2025. Features available in beta are still work-in-progress and may have bugs. We recognize the irony.
 
-If you have any questions, feedback or would like to report a bug, please open a [GitHub issue](https://github.com/getsentry/sentry-react-native/issues/new?assignees=&labels=Platform%3A+React-Native%2CType%3A+%F0%9F%AA%B2+Bug&projects=&template=BUG_REPORT.md) with a link to a relevant replay in Sentry if possible.
+All Beta users will continue to receive 2 months free of Session Replay for Mobile once we transition to GA.
```
Comment (drguthals) [inline docs/platforms/react-native/session-replay/index.mdx:12]:
```suggestion
All beta users will continue to receive 2 months free of Session Replay for Mobile once we transition to GA.
```

**Review by ryan953 — APPROVED** [review]

**Review by coolguyzone — APPROVED** [review]
Looks good! Thanks for updating 🕺

**Pull request merged**